src/cmd/flash.js

```diff
@@ -50,7 +50,8 @@ module.exports = class FlashCommand extends CLICommandBase {
 		if (usb) {
 			await this.flashOverUsb({ binary, factory });
 		} else if (serial) {
-			await this.flashYModem({ binary, port, yes });
+			const serialCmdInstance = new SerialCommand();
+			await serialCmdInstance.flashDevice(binary, { port, applicationOnly: true });
```

Comment:
Using serial command's flashDevice because we would like to have the port parsed. This is so we don't break any users' scripts or usages that use `--port` option if any